lib/Image/Adapter/Imagick.php

```diff
@@ -558,8 +558,8 @@ public function frame(int $width, int $height, bool $forceResize = false): stati
 
         $this->contain($width, $height, $forceResize);
 
-        $x = ($width - $this->getWidth()) / 2;
-        $y = ($height - $this->getHeight()) / 2;
+        $x = intval(($width - $this->getWidth()) / 2);
```

Comment:
According to a lot of best practice, intval is slower than just casting it to int.
I prefer casting directly instead of using intval, since intval is more common if you have a string that needs to be converted to int or you need another base (if you don't need base 10 e.g.)
```suggestion
        $x = (int)(($width - $this->getWidth()) / 2);
```